app/models/user.rb

```diff
@@ -66,7 +68,11 @@ def multiple_identities?
   end
 
   def active_profile
-    profiles.where(active: true).first
+    profiles.find(&:active?)
+  end
+
+  def verified?
+    active_profile ? true : false
   end
```

Comment:
Could we please keep these types of querying methods in the `UserDecorator`? We'd like to keep the User model lean and dealing only with persistence if possible.